ext/tiny_tds/client.h

```diff
@@ -5,9 +5,9 @@
 void init_tinytds_client();
 
 #define ERROR_MSG_SIZE 1024
+#define ERRORS_STACK_INIT_SIZE 2
```

Comment:
What's special about 2?

Reply:
Nothing really. I picked a value that wasn't too greedy and that would support most use cases (0-2 errors reported) without needing to reallocate any memory. I'd be open to changing this to something else.

Reply:
Makes sense